test/unit/postprocessing/test_threshold_optimization.py

```diff
@@ -150,24 +150,19 @@ def test_threshold_optimization_different_input_lengths(data_X_y_sf, constraints
                                    sensitive_features=data_X_y_sf.sensitive_features)
 
 
-@pytest.mark.parametrize("score_transform", candidate_Y_transforms)
 @pytest.mark.parametrize("y_transform", candidate_Y_transforms)
 @pytest.mark.parametrize("sensitive_features_transform", candidate_A_transforms)
-def test_threshold_optimization_demographic_parity(score_transform, y_transform,
+def test_threshold_optimization_demographic_parity(y_transform,
                                                    sensitive_features_transform):
     y = y_transform(labels_ex)
     sensitive_features = sensitive_features_transform(sensitive_features_ex1)
-    scores = score_transform(scores_ex)
```

Comment:
I'm fine with the other changes, but score_transform should still be used, otherwise we don't have test coverage for other cases.

Reply:
I don't think we need to vary score transform in this instance. The base estimator is supposed to be implementing an sklearn estimator and those are always supposed to return 1D-arrays (not lists and not DataFrames). I don't think we need to try to be more general than sklearn here.

Reply:
On a second thought, let me try to add this back. I don't trust people to implement sklearn estimators 100% faithfully to the spec, and it would be nice to cover these simple cases.

Reply:
Then we need to adjust the expected data types to exclude those. At every other point we've parametrized over the different data types.

Reply:
`score_ex` is now again included and I've also added a comment about the `PassThroughPredictor`